qutebrowser/browser/webengine/webview.py

```diff
@@ -238,3 +238,12 @@ def acceptNavigationRequest(self,
             is_main_frame=is_main_frame)
         self.navigation_request.emit(navigation)
         return navigation.accepted
+
+    def chooseFiles(self, mode, old_files, accepted_mimetypes):
+        """Override chooseFiles to (optionally) invoke custom file uploader."""
+        handler = config.val.fileselect.handler
+        if handler == "default":
```

Comment:
Nitpick: Simpler to use the config value directly here I'd say:

```suggestion
        if config.val.fileselect.handler == "default":
```

Reply:
Fixed manually